website/static/js/oauthAddonNodeConfig.js

```diff
@@ -0,0 +1,244 @@
+/**
+ * Module that controls the addon node settings. Includes Knockout view-model
+ * for syncing data, and HGrid-folderpicker for selecting a folder.
+ */
+'use strict';
+
+var ko = require('knockout');
+require('knockout.punches');
```

Comment:
As discussed privately, weird behavior happens with `selected` and `selectedFolderName` when removing, and a 4kb download that the user should already have isn't worth it.